assets/app/scripts/app.js

```diff
@@ -286,7 +286,7 @@ angular
       });
   })
   .constant("API_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).api))
-  .constant("APIS_CFG", angular.extend({}, (window.OPENSHIFT_CONFIG || {}).apis))
+  .constant("APIS_CFG", (window.OPENSHIFT_CONFIG || {}).apis)
```

Comment:
I think it was to scope down which bits of the config were needed by different parts